test/ufuzz.js

```diff
@@ -440,7 +502,7 @@ for (var round = 0; round < num_iterations; round++) {
     var beautify_result = run_code(beautify_code);
 
     try {
-      var uglify_code = minify(beautify_code, {
+      var uglify_code = minify(original_code, {
```

Comment:
I would expect it to test both the beautifier and the minifier this way. I assume they use completely different code paths after parsing. Parsing doesn't appear to be a bottleneck because the beautification is about as fast as I'd expect it and I expect that to use the same parser.

I don't mind either way, you have to use the tool so it's really up to you.